samples/snippets/src/main/java/com/example/storage/object/UploadEncryptedObject.java

```diff
@@ -48,10 +48,21 @@ public static void uploadEncryptedObject(
     Storage storage = StorageOptions.newBuilder().setProjectId(projectId).build().getService();
     BlobId blobId = BlobId.of(bucketName, objectName);
     BlobInfo blobInfo = BlobInfo.newBuilder(blobId).build();
+
+    // Optional: set a generation-match precondition to avoid potential race
+    // conditions and data corruptions. The request returns a 412 error if the
+    // preconditions are not met.
+    // For a target object that does not yet exist, set the DoesNotExist precondition.
+    Storage.BlobTargetOption precondition = Storage.BlobTargetOption.doesNotExist();
+    // If the destination already exists in your bucket, instead set a generation-match
+    // precondition:
+    // Storage.BlobTargetOption precondition = Storage.BlobTargetOption.generationMatch();
+
     storage.create(
         blobInfo,
         Files.readAllBytes(Paths.get(filePath)),
-        Storage.BlobTargetOption.encryptionKey(encryptionKey));
+        Storage.BlobTargetOption.encryptionKey(encryptionKey),
+        precondition);
```

Comment:
(nit) Move this to the same line (L64)

Reply:
[Owlbot did this](https://github.com/googleapis/java-storage/pull/1600/commits/4eded73d48a6c47afea6a6f16776f4d5cb048273#diff-9ad60db022b5fd58482b3e639195b3b8dd7d6d320020ff82739d2506b96ac834R57) and will just change it back if I try to fix it